dvc/repo/experiments/queue/celery.py

```diff
@@ -401,8 +404,8 @@ def logs(
             {rev}, self.iter_active(), self.iter_done()
         ).get(rev)
         if queue_entry is None:
-            if rev in self.match_queue_entry_by_name(
-                {rev}, self.iter_queued()
+            if self.match_queue_entry_by_name({rev}, self.iter_queued()).get(
+                rev
```

Comment:
@karajan1001 This was always true since it returns something like `{"my-exp": None}`, so it was causing any invalid name to return `Experiment '{rev}' is in queue but has not been started`.

Reply:
Maybe we can just remove it, as it's the same as above. Only if the experiments get updated between these two phrases will make they return differently.

Reply:
I thought above was checking if the rev is active, and this is checking whether the inactive rev is in the queue at all?

Reply:
Otherwise, you will get a message that it's not a valid experiment name even for valid but inactive experiments, right?

Reply:
OK,  I missed the difference of iter_queued and iter_active.